docs/topics/signals.rst

```diff
@@ -114,7 +114,7 @@ engine_started
 
     Sent when the Scrapy engine has started crawling.
 
-    This signal supports returning deferreds from its handlers.
+    This signal supports asynchronous handlers.
```

Comment:
This may be a good chance to turn “asynchronous handlers” into links to the corresponding section above.